package.json

```diff
@@ -52,6 +52,10 @@
     "generate:example": "node bin/example-generator/index.js",
     "generate:docs": "rimraf docs/ && jsduck --config jsduck.json",
     "generate:docs-w-ext": "rimraf docs-w-ext/ && jsduck --config jsduck-w-ext.json",
+    "generate:docs:master": "jsduck --config=jsduck.json --output=master/docs --title=\"GeoExt $npm_package_version-dev Documentation\" --warnings=-inheritdoc",
+    "generate:docs-w-ext:master": "jsduck --config=jsduck.json --output=master/docs-w-ext --title=\"GeoExt $npm_package_version-dev Documentation (incl. ExtJS classes)\" --warnings=-all:resources/external/ext-6.2.0",
```

Comment:
`--config=jsduck.json` should be `--config=jsduck-w-ext.json`

Reply:
`--warnings=…` should be moved to configfile `jsduck-w-ext.json`

Reply:
Don't we need to also point to the source files of an ExtJS folder? in `jsduck-w-ext.json` this is done by pointing to 

```
        "../ext-6.2.0/packages/core/src/",
        "../ext-6.2.0/classic/classic/src/"
```

This folders of course need to exist and shouldprobably be fetched during `install` and please also cached.

Reply:
Fixed in 6a24509b